pom.xml

```diff
@@ -67,14 +69,31 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>analysis-core</artifactId>
-            <version>1.71</version>
-            <type>hpi</type>
+            <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
It seems that a bug in trend graphs were fixed in analysis-core 1.74. I think it's safe to upgrade to 1.74 since there are no big changes (only a bug fix).
Thanks Bruno!